src/Stratis.Bitcoin.IntegrationTests/Wallet/WalletTestsManyTxnInputsAndOutputsSpecification.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.Wallet
+{
+    public partial class WalletTestsManyTxnInputsAndOutputsSpecification 
+    {
+        [Fact]
+        public void Wallet_can_send_one_transaction_with_many_outputs()
+        {
+            Given(a_stratis_sender_and_receiver_node_and_their_wallets);
+            And(a_block_is_mined);
+            When(a_transaction_is_sent_to_receiver_via_fifty_outputs);
+            Then(the_transaction_is_recevied);
+        }
+
+        [Fact]
+        public void Wallet_can_send_and_receive_many_transaction_inputs_and_ouputs()
```

Comment:
In this situation where the first 4 lines of this test are the FULL previous test,  I think it makes more sense to just follow on the first test and do another When Then..

eg
Given(a_stratis_sender_and_receiver_node_and_their_wallets);
And(a_block_is_mined);
When(a_transaction_is_sent_to_receiver_via_fifty_outputs);
Then(the_transaction_is_received);
When(the_receiver_sends_back_from_those_fifty_outputs)
Then(the_transaction_is_received_by_sender);

Now it gets confusing - so perhaps node1 and node2 or nodeA and nodeB might be clearer

Reply:
Is "transaction received" the right domain language?  is it more like the "funds are received" or "balance is updated". I am thinking because the transaction is the thing on the block chain. @dangershony thoughts?